src/kontrol/kdist/cheatcodes.md

```diff
@@ -907,6 +913,48 @@ Otherwise, throw an error for any other call to the Foundry contract.
       [owise]
 ```
 
+
```

Comment:
Please add a description of the cheat code, including the current limitations that it has.

Reply:
I forgot to do that. Thanks!